FirebaseAppDistributionInternal/Sources/AppDistributionApiService.swift

```diff
@@ -86,17 +104,29 @@ enum AppDistributionApiError: NSInteger {
   }
 
   @objc(fetchReleasesWithCompletion:) public static func fetchReleases(completion: @escaping AppDistributionFetchReleasesCompletion) {
+    fetchReleases(
+      app: FirebaseApp.app()!,
```

Comment:
Should we use a guard here instead of a forced unwrapping?

Reply:
Done